.rustfmt.toml

```diff
@@ -0,0 +1,24 @@
+# stable options
+use_small_heuristics = "Default"
+force_explicit_abi = true
+hard_tabs = false
+max_width = 100
+merge_derives = false
+newline_style = "Auto"
+remove_nested_parens = true
+reorder_imports = true
+reorder_modules = true
+tab_spaces = 4
+use_field_init_shorthand = true
+use_try_shorthand = true
+edition = "2018"
+
+# unstable options (for one-off fixes and CI)
+merge_imports = true
```

Comment:
The default is `false` but I changed this to `true` because I think we've all agreed that this is a style we want to move towards and having all of that work done for us for free is amazing.
https://github.com/rust-lang/rustfmt/blob/master/Configurations.md#merge_imports